parsers/US_PJM.py

```diff
@@ -115,6 +119,50 @@ def data_processer(data):
     return production
 
 
+def fetch_consumption_forecast(zone_key='US-PJM', session=None,
+                               target_datetime=None, logger=None):
+    """
+    Gets consumption forecast for specified zone.
+    Returns a list of dictionaries.
+    """
+
+    if target_datetime:
+        raise NotImplementedError(
+            'This parser is not yet able to parse past dates')
+    if not session:
+        session = requests.session()
+
+    if 'PJM_TOKEN' not in os.environ:
+        raise Exception('No PJM_TOKEN found! Please add it to secrets.env!')
+    headers = {'Ocp-Apim-Subscription-Key': os.environ['PJM_TOKEN']}
+
+    # startRow must be set if forecast_area is set.
+    # RTO_COMBINED is area for whole PJM zone.
+    params = {
+        'download': True,
+        'startRow': 1,
+        'forecast_area': 'RTO_COMBINED'
+    }
+
+    # query API
+    url = API_ENDPOINT + 'load_frcstd_7_day'
+    resp = requests.get(url, params, headers=headers)
+    data = json.loads(resp.content)
+
+    data_points = list()
+    for elem in data:
+        utc_datetime = elem['forecast_datetime_beginning_utc']
+        data_point = {
+            'zoneKey': zone_key,
+            'datetime': arrow.get(utc_datetime).replace(tzinfo='UTC').datetime,
+            'value': elem['forecast_load_mw'],
+            'source': 'pjm.com'
+        }
+        data_points.append(data_point)
+
+    return data_points
```

Comment:
Can you post the output of the parser.

Reply:
sample output [attached](https://github.com/tmrowco/electricitymap/files/2103816/pjm_output.txt)